lib/viewmodels/new_comment_view_model.dart

```diff
@@ -82,7 +86,21 @@ class NewCommentViewModel
       voteScore: 0,
     );
 
-    await commentRepository.addComment(postId, commentData);
+    // Note: The addition of the comment unedr the post and the comment under the
```

Comment:
Small typo:
```suggestion
    // Note: The addition of the comment under the post and the comment under the
```

Reply:
Thanks !

Fix in 643b7788b53fa6b6b1e97629276deb3208cbd70a.